### PR TITLE
Nicolle fix team member tasks

### PR DESCRIPTION
**Pull request by nicollext**
# Description
image

## Mainly changes explained:
A container with an overflow property set to "auto" has been added to enable users to scroll through the Team member task table on smaller screens

## How to test:
check into current branch
do `npm install` and `...` to run this PR locally
go to dashboard
Check if the table is being displayed correctly.

## Screenshots or videos of changes:

**Before**

https://user-images.githubusercontent.com/101591708/234395863-2f5ed79c-4f34-44dc-bdcc-9fe2ccad18c3.MP4

.....
**After**

https://user-images.githubusercontent.com/101591708/234395877-34fd3ec0-03d2-45e6-9507-dc0273a4c633.MP4

### CODE REVIEW

**Review by AuroraLovesWhales — APPROVED** [review]
Hi Nicolle, I have tested your PR and the TeamMemberTasks component is looking great on a smaller screen now!

Before:
Pasted Graphic 10

After:
Pasted Graphic 12
https://user-images.githubusercontent.com/44281916/234406362-6a70d3fc-7fca-45ce-8b04-aefb13ba8a9d.mov

**Comment by AnishPandita** [comment]
Hi @nicollext I tested your PR and it seems to be working great. The container overflow seems to be solved.

https://user-images.githubusercontent.com/111321945/234424960-d1f789b9-a749-4206-82e7-aee424482047.mov

File: src/components/TeamMemberTasks/style.css
Comment (Vishvesh-Sheoran) [inline src/components/TeamMemberTasks/style.css:1]: That's how we can use style.css in the teamMemberTaks.jsx file above:

"import React from 'react';
import { Table } from 'reactstrap';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { faClock } from '@fortawesome/free-solid-svg-icons';
import Loading from './Loading';
import SaveButton from './SaveButton';

import './styles/TeamMemberTasks.css';

const TeamMemberTasks = ({
  isLoading,
  teamMembers,
  handleClockToggle,
  userRole,
  setClickedToShowModal,
}) => {

  const renderTeamsList = () => {
    return teamMembers.map((teamMember) => {
      return (
        <tr key={teamMember._id}>
          <td className='team-member-tasks-user-profile'>
            {/* User profile and tasks */}
          </td>
          <td className='team-member-tasks-timesheet'>
            {/* Timesheet */}
          </td>
          <td className='team-member-tasks-tasks'>
            {/* Task(s) */}
          </td>
        </tr>
      );
    });
  };

  return (
    <div className='team-member-tasks'>
      <h1>Team Members</h1>
      {userRole === 'Administrator' && (
        <SaveButton
          className='team-member-tasks-bell'
          handleSubmit={() => console.log('Saving changes...')}
        />
      )}
      <div className='table-container'>
        <Table>
          <thead className='pc-component'>
            <tr>
              {/* Empty column header for hours completed icon */}
              <th />
              <th className='team-member-tasks-headers'>
                <Table borderless className='team-member-tasks-subtable'>
                  <thead>
                    <tr>
                      <th className='team-member-tasks-headers team-member-tasks-user-name'>
                        Team Member
                      </th>
                      <th className='team-member-tasks-headers team-clocks team-clocks-header'>
                        <FontAwesomeIcon icon={faClock} title='Weekly Committed Hours' />
                        /
                        <FontAwesomeIcon
                          style={{ color: 'green' }}
                          icon={faClock}
                          title='Total Hours Completed this Week'
                        />
                        /
                        <FontAwesomeIcon
                          style={{ color: 'red' }}
                          icon={faClock}
                          title='Total Remaining Hours'
                        />
                      </th>
                    </tr>
                  </thead>
                </Table>
              </th>
              <th className='team-member-tasks-headers'>
                <Table borderless className='team-member-tasks-subtable'>
                  <thead>
                    <tr>
                      <th>Tasks(s)</th>
                      <th className='team-task-progress'>Progress</th>
                      {userRole === 'Administrator' ? <th>Status</th> : null}
                    </tr>
                  </thead>
                </Table>
              </th>
            </tr>
          </thead>

          <tbody>{isLoading ? <Loading /> : renderTeamsList()}</tbody>
        </Table>
      </div>
    </div>
  );
};"

Comment (nicollext) [reply src/components/TeamMemberTasks/style.css:1]: Hello, can you provide additional information on this matter? I'm having difficulty comprehending what you're referring to.

**Review by abdel-lall — APPROVED** [review]
Works as expected. good job!

PR791

**Review by alexander-huerta — APPROVED** [review]
Looks good! 

I tested these changes by changing the window width in the browser back and forth, using both Chrome and Brave. The overflow has been resolved without creating any other style or functionality issues.

**Comment by GKD890** [comment]
Not sure if Safari is in our support list, but tried that comparing with Chrome. If not, please ignore this. 

During resizing window (particularly width) from large to small, seems the `member task list` won't fold even in small size window, and `leaderboard` will go to bottom under the `member task list`  from top at the end.

https://www.dropbox.com/s/uarh2k7rj19vuda/Frontend_PR%23791.mov?dl=0 if you are curious in Safari.

**Pull request merged**